dindex_store/discovery_index.py

```diff
@@ -122,6 +122,8 @@ def get_minhashes(self) -> Dict:
 
     def find_neighborhood(self, node_id: int, relation_type, hops: int = 1, desired_attributes=None):
         neighbors_ids = self.__graph_index.find_neighborhood(node_id, relation_type, hops)
+        if len(neighbors_ids) == 0:
+            return []
```

Comment:
I note that below (line 129) we return None while here we return an empty list. I'm not sure which one we want to return.

Reply:
ummm, I don't think we return None anywhere in this function. Line 129 fetches profiles for columns in `neighbors_ids`. line 125-126 is to deal with the corner case when the input node has no neighbors